lib/ledger_sync/adaptors/netsuite/account/ledger_deserializer.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Account
+        class LedgerDeserializer < NetSuite::LedgerSerializer
+          attribute ledger_attribute: :id,
+                    resource_attribute: :ledger_id
+
+          attribute ledger_attribute: :acctname,
+                    resource_attribute: :name
```

Comment:
While we're here, are there other attributes we can put in here? Similar to the attributes we have for account

Or do they get filled in automagically (I don't recall how that works)